### PR TITLE
Simpler APIs

### PR DESCRIPTION
**Pull request by hiroshihorie**

### CODE REVIEW

File: lib/src/types.dart [hunk 1]
```diff
@@ -29,6 +29,13 @@ enum Reliability {
   lossy,
 }
 
+enum TrackSource {
+  unknown,
+  camera,
+  microphone,
+  screenShare,
```
Comment (davidzhao) [inline lib/src/types.dart:36]: FYI: I also added a ScreenShareAudio type.. for when the screen share is shared with an audio track.

**Review by davidzhao — COMMENTED** [review]
lg, just a minor comment

File: lib/src/participant/local_participant.dart [hunk 2]
```diff
@@ -230,3 +232,46 @@ class LocalParticipant extends Participant {
     super.updateFromInfo(info);
   }
 }
+
+extension LocalParticipantTrackSourceExt on LocalParticipant {
+  Future<void> setCamercaEnabled(bool enabled) async {
```
Comment (davidliu) [inline lib/src/participant/local_participant.dart:237]: nit: typo in `setCamercaEnabled` (should be `setCameraEnabled`)

**Review by davidliu — APPROVED** [review]

File: lib/src/participant/local_participant.dart (hunk 2 above)
Comment (hiroshihorie) [reply lib/src/participant/local_participant.dart:237]: Omg yesss thanks onichan!!! 🙏🏻😁

**Pull request merged**